build.gradle

```diff
@@ -289,6 +289,7 @@ dependencies {
 
     compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.10'
     compile group: 'uk.gov.service.notify', name: 'notifications-java-client', version: '3.12.0-RELEASE'
+    compile group: 'com.launchdarkly', name: 'launchdarkly-java-server-sdk', version: '4.12.1'
```

Comment:
it is there already: https://github.com/hmcts/ia-case-api/blob/master/build.gradle#L298

Reply:
Removed